### PR TITLE
Stop using jQuery plugin for timestamps

### PR DESCRIPTION
**Pull request by nt1m**

### CODE REVIEW

**Comment by bnjbvr** [comment]
Sorry I'll be on PTO for two weeks and won't be able to get this review quickly.

File: lib/routes.js [hunk 1]
```diff
@@ -80,7 +80,7 @@ exports.landingPage = function (response, user = null) {
     user,
     scripts: [
       '/js/landing.js',
-      '/js/jquery.timeago-1.5.4.min.js',
+      '/js/timeago.min.js',
```
Comment (jankeromnes) [inline lib/routes.js]: Nit: Please suffix third-party minified scripts with their versions where applicable.

File: static/js/projects.js [hunk 2]
```diff
@@ -42,13 +42,17 @@ Array.map(document.querySelectorAll('*[data-status]'), function (element) {
...
-  element.title = date.toUTCString().replace('GMT', 'UTC');
-  element.setAttribute('datetime', date.toISOString());
-
-  // Use jQuery's live-updating timeago plugin.
-  $(element).timeago();
+  try {
+    // GMT is deprecated (see https://en.wikipedia.org/wiki/UTC).
+    element.title = date.toUTCString().replace('GMT', 'UTC');
+    element.setAttribute('datetime', date.toISOString());
+  } catch (error) {}
```
Comment (jankeromnes) [inline static/js/projects.js]: Nit: Why do you need a try/catch here? Also, please never ever use `catch (error) {}` (instead, we should handle errors, e.g. log or alert them (or if we really want to ignore a specific failure, mention this in a comment, and only ignore specific error types or messages).

File: static/js/projects.js [hunk 3]
```diff
@@ -42,13 +42,17 @@ Array.map(document.querySelectorAll('*[data-status]'), function (element) {
 
 // Add fuzzy timestamps to elements with a 'data-timestamp' attribute.
 
-Array.map(document.querySelectorAll('*[data-timestamp]'), function (element) {
+var timestampElements = document.querySelectorAll('[data-timestamp]');
+Array.map(timestampElements, function (element) {
```
Comment (jankeromnes) [inline static/js/projects.js]: Nit: `Array.forEach` is probably more relevant here.

File: static/js/projects.js [hunk 4]
```diff
@@ -42,13 +42,17 @@ Array.map(document.querySelectorAll('*[data-status]'), function (element) {
 
 // Add fuzzy timestamps to elements with a 'data-timestamp' attribute.
```
Comment (jankeromnes) [inline static/js/projects.js]: Nit: Please drive-by remove the empty line between function comment and definition (which is an older coding style).

File: static/js/projects.js [hunk 5]
```diff
@@ -41,14 +41,14 @@ Array.map(document.querySelectorAll('*[data-status]'), function (element) {
...
   // GMT is deprecated (see https://en.wikipedia.org/wiki/UTC).
   element.title = date.toUTCString().replace('GMT', 'UTC');
   element.setAttribute('datetime', date.toISOString());
-
-  // Use jQuery's live-updating timeago plugin.
-  $(element).timeago();
 });
+
+// Use live-updating timeago plugin.
+timeago().render(timestampElements);
```
Comment (jankeromnes) [inline static/js/projects.js]: Nit: Why move `timeago().render(element)` out of the `forEach` loop? I don't think you need to do that, nor to create a variable for the collection.
